[2024-03-22T19:07:05.732+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: milestone3.clean_data scheduled__2024-03-22T19:06:00+00:00 [queued]>
[2024-03-22T19:07:05.746+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: milestone3.clean_data scheduled__2024-03-22T19:06:00+00:00 [queued]>
[2024-03-22T19:07:05.746+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-03-22T19:07:05.746+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2024-03-22T19:07:05.747+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-03-22T19:07:05.757+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): clean_data> on 2024-03-22 19:06:00+00:00
[2024-03-22T19:07:05.770+0000] {standard_task_runner.py:52} INFO - Started process 2329 to run task
[2024-03-22T19:07:05.773+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'milestone3', 'clean_data', 'scheduled__2024-03-22T19:06:00+00:00', '--job-id', '284', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_elia_DAG.py', '--cfg-path', '/tmp/tmpudija1ai', '--error-file', '/tmp/tmph6a1e7i_']
[2024-03-22T19:07:05.774+0000] {standard_task_runner.py:80} INFO - Job 284: Subtask clean_data
[2024-03-22T19:07:05.850+0000] {task_command.py:371} INFO - Running <TaskInstance: milestone3.clean_data scheduled__2024-03-22T19:06:00+00:00 [running]> on host b79cb602ca97
[2024-03-22T19:07:05.907+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=elia
AIRFLOW_CTX_DAG_ID=milestone3
AIRFLOW_CTX_TASK_ID=clean_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-22T19:06:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-22T19:06:00+00:00
[2024-03-22T19:07:06.256+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-03-22T19:07:06.283+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=milestone3, task_id=clean_data, execution_date=20240322T190600, start_date=20240322T190705, end_date=20240322T190706
[2024-03-22T19:07:06.323+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-03-22T19:07:06.370+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
